packages/contracts/contracts/voting/addresslist/AddresslistVoting.sol

```diff
@@ -115,7 +122,7 @@ contract AddresslistVoting is MajorityVotingBase {
     ) external override returns (uint256 proposalId) {
         uint64 snapshotBlock = getBlockNumber64() - 1;
 
-        if (!isListed(_msgSender(), snapshotBlock)) {
+        if (!(minProposerVotingPower == 0) && !isListed(_msgSender(), snapshotBlock)) {
```

Comment:
This check should be fine. `minProposerVotingPower` can be set `>1` although this don't makes much sense. The `votingPower` is always 1

Reply:
I don't get why we use both. can't we remove `minProposerVotingPower==0` check ?